### PR TITLE
Create ChangeManager component

### PR DESCRIPTION
**Pull request by Igor-Gorchakov**
- Create ChangeManager component
- Stub REST method for JobExecution entities initialization

testpostmanrq

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -17,6 +17,10 @@
         {
           "methods": ["GET"],
           "pathPattern": "/metadata-provider/jobExecutions"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/change-manager/jobExecutions"
```
Comment (OleksiiKuzminov) [inline descriptors/ModuleDescriptor-template.json]: perms?

File: ramls/dto/initJobExecutionsRqDto.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Request to initialize JobExecution entities",
+  "additionalProperties": false,
+  "properties": {
+    "uploadDefinitionId": {
```
Comment (OleksiiKuzminov) [inline ramls/dto/initJobExecutionsRqDto.json]: we dont need relations on data-import module
just 
{
files:[
{name: "marc.bib"},
{name: "test.mrc"}
]
}

File: ramls/dto/initJobExecutionsRsDto.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Initialized JobExecution entities",
+  "additionalProperties": false,
+  "properties": {
+    "uploadDefinitionId": {
```
Comment (OleksiiKuzminov) [inline ramls/dto/initJobExecutionsRsDto.json]: dont need this field

File: src/test/java/org/folio/rest/impl/ChangeManagerAPITest.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+  // TODO reuse verticle deployment
+  @BeforeClass
+  public static void setUpClass(final TestContext context) throws Exception {
+    Async async = context.async();
+    vertx = Vertx.vertx();
+    port = NetworkUtils.nextFreePort();
+
+    useExternalDatabase = System.getProperty(
+      "org.folio.password.validator.test.database",
```
[the remark this answers is not in the document]
Comment (Igor-Gorchakov) [reply src/test/java/org/folio/rest/impl/ChangeManagerAPITest.java]: agree, fixed

File: src/test/java/org/folio/rest/impl/ChangeManagerAPITest.java [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+  private static final String CHANGE_MANAGER_PATH = "/change-manager";
+  private static final String ACCEPT_VALUES = "application/json, text/plain";
+
+  private static Vertx vertx;
+  private static int port;
+  private static String useExternalDatabase;
+
+  // TODO reuse verticle deployment
+  @BeforeClass
+  public static void setUpClass(final TestContext context) throws Exception {
```
[the remark this answers is not in the document]
Comment (Igor-Gorchakov) [reply src/test/java/org/folio/rest/impl/ChangeManagerAPITest.java]: agree, fixed

File: NEWS.md [hunk 6]
```diff
@@ -1,3 +1,13 @@
+## 2018-11-19 v1.0.2
+ * Created ChangeManager component
+ * Added REST method for JobExecution entities initialization
+ * REST method has stub implementation
```
[the remark this answers is not in the document]
Comment (Igor-Gorchakov) [reply NEWS.md]: agree, fixed

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (Igor-Gorchakov) [reply descriptors/ModuleDescriptor-template.json]: agree

**Pull request merged**